vm/message/src/unsigned_message.rs

```diff
@@ -288,7 +290,8 @@ pub mod json {
         sequence: u64,
         #[serde(with = "bigint_ser::json")]
         value: TokenAmount,
-        gas_limit: i64,
+        #[serde(with = "bigint_ser::json::gas")]
+        gas_limit: Gas,
```

Comment:
The `gas_limit` in a `Message` is always an `i64` so we don't need to store it as `Gas`.

Reply:
Fixed